Mayhemfile

```diff
@@ -0,0 +1,5 @@
+project: chrono
+target: chrono
```

Comment:
As with other repos, please set the target name to reflect the fuzz target (`fuzz_reader` in this case).